### PR TITLE
Updated redis image to 4.0.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
`Diffing package redis revision 1 revision 2


Diffing File config.json:


Diffing File marathon.json.mustache:


Diffing File package.json:
4c4
<  "version": "3.2.9-0.0.1",
---
>  "version": "4.0-0.0.1",
6c6
<  "description": "This is a single redis container, which is suited for HA setups. Redis is a popular in-memory data structure store, used as database, cache and message broker.. See documentation for more details: https://github.com/dcos/examples/tree/master/redis ",
---
>  "description": "This is a single redis container, which is NOT suited for HA setups. Redis is a popular in-memory data structure store, used as database, cache and message broker. See documentation for more details: https://github.com/dcos/examples/tree/master/redis",
18c18
<  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Redis requires a single node with 300MB of RAM and 0.1 CPUs. ",
---
>  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Redis requires a single node with 300MB of RAM and 0.1 CPUs.",


Diffing File resource.json:
13c13
<        "redis": "redis:3.2.9"
---
>        "redis": "redis:4.0"`

**Comment by judithpatudith** [comment]
👍

File: repo/packages/R/redis/2/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ "tags": ["mesosphere", "service", "redis", "storage"],
+ "licenses": [
+   {
+     "name": "BSD",
+     "url": "http://redis.io/topics/license"
+   }
+ ],
+ "website": "http://redis.io",
+ "postInstallNotes": "Redis installed!",
+ "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Redis requires a single node with 300MB of RAM and 0.1 CPUs.",
```
Comment (benclarkwood) [inline repo/packages/R/redis/2/package.json]: This is incorrect. 1 cpu, 1024 MB according to config.json.

Comment (joerg84) [reply repo/packages/R/redis/2/package.json:18]: You can still configure less.

**Review by cprovencher — APPROVED** [review]

**Pull request merged**